### ISSUE TITLE
Provide clear documentation or possibility (if does not exist yet) on usage of pem file with both certificate and private key file inside

### ISSUE THREAD

**Issue by oleksiilv** [cherrypy/cherrypy#1802]
**I'm submitting a ...**
- [ ] bug report
- [x] feature request
- [ ] question about the decisions made in the repository

**Do you want to request a *feature* or report a *bug*?**
*feature*

**What is the current behavior?**
not documented, unspecified.
It seems to work if we specify the following, but it is not clear whether it is recommended and works as expected:
```python
cherrypy.server.ssl_module = 'builtin'
cherrypy.server.ssl_certificate = pem_certificate_with_certificate_and_private_key
cherrypy.server.ssl_private_key = pem_certificate_with_certificate_and_private_key
```

or even just:
```python
cherrypy.server.ssl_module = 'builtin'
cherrypy.server.ssl_certificate = pem_certificate_with_certificate_and_private_key
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a screenshots and logs of the problem. If you can, show us your code.**

N/A

**What is the expected behavior?**

possibility to use pem file with both certificate and private inside is specified and documented

**What is the motivation / use case for changing the behavior?**

Security is very important for web applications so it is better to have proofs from documentation that what is used is correct.
Developers are asking about such behavior, e.g.:
https://github.com/cherrypy/cherrypy/issues/1563#issuecomment-280045357

**Please tell us about your environment:**

- CherryPy version: 18.1.2
- Python version: 3.6.5
- OS: any
- Browser: [all]

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, e.g. stackoverflow, gitter, etc.)

N/A

**Comment by webknjaz** [comment]
Yeah, it's used like this even in tests too: https://github.com/cherrypy/cherrypy/blob/db0a8a6/cherrypy/test/helper.py#L435-L439

The code for TLS adapters is @ https://github.com/cherrypy/cheroot/tree/e03c3f5d362a7ac5d2517aabbc7b60b10fd83d1d/cheroot/ssl. CherryPy mostly just maps config values to their initializers.

If you have ideas on the docs improvements, feel free to send a PR against https://github.com/cherrypy/cherrypy/blob/db0a8a6/docs/deploy.rst#L174-L237
